<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Вступление</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>631</width>
        <height>311</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Привет!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;И &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Добро Пожаловать&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; на тестирование знаний по спортивной тематике! &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Тест пробегается на время, поэтому после нажатия кнопки&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;&amp;quot;Старт&amp;quot; &lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;будет включен таймер, по окончании которого тест финишируется автоматически (DNF). До финиша можно добраться быстрее таймера нажатием на кнопку &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;&amp;quot;Финишировать тест&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;, после чего ты узнаешь, насколько ты аналитик МАТЧ ТВ! Встретимся на финише!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lemon/Milk(RUS BY LYAJKA)</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Какова длина самой большой дистанции &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;по бегу в легкой атлетике - марафона?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1_1">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>42.195 км</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>41.195 км</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>42 км</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_1_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>300</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>43 км</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1_accept">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>340</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердить ответ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>5</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Результат</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>121</width>
        <height>351</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Ответ</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_finish">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Финишировать тест</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
